README.md

```diff
@@ -5,23 +5,24 @@
 
 [Documentation](https://tonybaloney.github.io/CSnakes/)
 
-[![NuGet Version](https://img.shields.io/nuget/v/CSnakes?label=CSnakes)](https://www.nuget.org/packages/CSnakes)
 [![NuGet Version](https://img.shields.io/nuget/v/CSnakes.Runtime?label=CSnakes.Runtime)](https://www.nuget.org/packages/CSnakes.Runtime)
 
 <img src="docs/res/logo.jpeg" alt="drawing" width="200"/> 
 
 CSnakes is a .NET Source Generator and Runtime that you can use to embed Python code and libraries into your .NET Solution without the need for REST, HTTP, or Microservices.
 
-![image](https://github.com/tonybaloney/PythonCodeGen/assets/1532417/39ca2f2a-416b-447a-a237-59e9613a4990)
+![image](https://github.com/tonybaloney/CSnakes/assets/1532417/39ca2f2a-416b-447a-a237-59e9613a4990)
 
 ## Features
 
-- .NET Standard 2.0 (.NET 6-8)
+- .NET Standard 2.0 (.NET 6-9+)
```

Comment:
```suggestion
- .NET Standard 2.0 (.NET 8-9+)
```

Actually, I'm not sure we're .NET Standard 2.0, I think only the source generator is.

Reply:
Good point